devicetypes/smartthings/bose-soundtouch.src/bose-soundtouch.groovy

```diff
@@ -28,6 +28,8 @@ metadata {
         capability "Refresh"
         capability "Music Player"
         capability "Health Check"
+		capability "Sensor"
```

Comment:
**TABS** not *spaces* guys! TABS not spaces! (OK... I know that's a ***religious issue***; but, maaaan... Tabs take less filespace, and are automatically adjustable to the developer's preference (2 spaces vs 3 or 4). My indentations are exactly 2 tabs (instead of 8 spaces).

Any of y'all watch Silicon Valley TV series? Guy lost his girlfriend over this issue! :roll_eyes: